modules/pubmaticBidAdapter.js

```diff
@@ -257,6 +361,10 @@ function _createImpressionObject(bid, conf) {
     }
 
     impObj.video = videoObj;
+  } else if (bid.nativeParams) {
+    impObj.native = {};
+    var nativeRequest = _createNativeRequest(bid.nativeParams)
```

Comment:
avoid the declaration of a short-lived variable, nativeRequest

Reply:
Done 
Removed short lived variable